### PR TITLE
Fix unmanaged library default search

### PR DESCRIPTION
**Pull request by X9VoiD**
Thanks to @AbstractQbit for debugging this issue.

### CODE REVIEW

File: OpenTabletDriver/PluginContext.cs [hunk 1]
```diff
@@ -47,27 +47,18 @@ protected override Assembly Load(AssemblyName assemblyName)
...
-            if (baseLoad != IntPtr.Zero)
-                return baseLoad;
-
-            // Default behaviour failed, search <plugin>/runtimes for the dll
             if (this.PluginPath == null)
             {
-                Log.Write("Plugin", $"Obsolete plugin does not support loading native library '{unmanagedDllName}'", LogLevel.Warning);
-                return IntPtr.Zero;
+                Log.Write("Plugin", $"Independent plugin does not support loading native library '{unmanagedDllName}'", LogLevel.Warning);
+                throw new NotImplementedException();
```
Comment (InfinityGhost) [inline OpenTabletDriver/PluginContext.cs:53]:
```cs
throw new NotSupportedException();
```

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver/PluginContext.cs (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver/PluginContext.cs]: Requested change pushed

File: OpenTabletDriver/PluginContext.cs [hunk 2]
```diff
@@ -47,27 +47,18 @@ protected override Assembly Load(AssemblyName assemblyName)
...
+                throw new NotSupportedException();
             }
 
-            var libraryFile = Directory.EnumerateFiles(Path.Join(this.PluginPath, "runtimes"), ToDllName(unmanagedDllName), SearchOption.AllDirectories).FirstOrDefault();
-            try
-            {
+            var runtimeFolder = new DirectoryInfo(Path.Join(this.PluginPath, "runtimes"));
+            var libraryFile = runtimeFolder.Exists ? Directory.EnumerateFiles(runtimeFolder.FullName, ToDllName(unmanagedDllName), SearchOption.AllDirectories).FirstOrDefault() : null;
+            if (!string.IsNullOrEmpty(libraryFile))
                 return LoadUnmanagedDllFromPath(libraryFile);
```
Comment (InfinityGhost) [inline OpenTabletDriver/PluginContext.cs:59]: Does this intend to call `base.LoadUnmanagedDllFromPath()` or `this.LoadUnmanagedDllFromPath()`

Comment (X9VoiD) [reply OpenTabletDriver/PluginContext.cs:59]: `LoadUnmanagedDllFromPath` cannot be overriden, so this line calls `base.LoadUnmanagedDllFromPath()`

**Review by InfinityGhost — APPROVED** [review]

**Review by InfinityGhost — APPROVED** [review]

**Comment by InfinityGhost** [comment]
lol approved twice

**Pull request merged**